### ISSUE TITLE
Path errors are ambiguous

### ISSUE THREAD

**Issue by wyqydsyq** [curtiswilkinson/zoey-cli#26]
If you specify some rooms with invalid paths, you get a generic error:
```
ROOM        STATUS
1           Error 
2           Error 

Okay damon, all done! it took me 0 seconds to do everything

It looks like there was a few errors...
If I was you, I would go run the build commands you provided in the room and see what happens!
```
These two rooms had `path` properties that were incorrect (non-existent directories) but this error message misleads you into thinking it's a problem in the build commands themselves

**Comment by curtiswilkinson** [comment]
Makes sense! I might write a little "validate config" function that is called at the start of every build to validate paths exist and the config is correctly shaped

Are you going to be using roomservice much? Let me know if there is any other config pitfalls that you think should be validated
